.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -1,34 +1,11 @@
 ---
 name: Feature Request
 about: Suggest a new feature or enhancement for the APE library
-title: '[FEATURE] '
+title: ''
 labels: 'enhancement'
-assignees: vedran-kasalica
+assignees: ''
 
 ---
 
-**Note:** Not all fields are obligatory, but the more details you provide, the easier it will be for us to address the issue. Feel free to omit sections that you feel are not relevant to your report.
-
 ## Feature Description
 <!-- Provide a clear and concise description of the feature or enhancement you're proposing. -->
```

Comment:
Perhaps it would be useful to query the OP for the location of enhancement (subsystem)